.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: Release
+
+on:
+  push:
```

Comment:
Should the release workflow be triggered on a GitHub release? I see below the publish workflow only runs if `startsWith(github.ref, 'refs/tags/')`

Reply:
Creating a tag is a `push` action for GitHub, just with a special `github.ref` pointing to the tag. A tag is created when you publish a release, so this is equivalent for our purposes.

I think it's also safer, cause I've run into scenarios where the workflow triggered by publishing a release doesn't correctly set `github.ref` to the tag probably because of a race condition on GitHub's end, and the job naturally fails.

Reply:
This is what we do in the SDK fwiw: https://github.com/meltano/sdk/blob/b5ed8259dbcd5987b6440da60e1bb64d5a218471/.github/workflows/release.yml#L17